### PR TITLE
Removed related locations

### PR DESCRIPTION
**Pull request by orangejenny**
##### SUMMARY
https://github.com/dimagi/commcare-hq/pull/26646 required updating related locations, which wasn't productive, so I'm looking at killing it.

This was experimental for REACH. It's only turned on for a handful of domains, all on india, all of which are internally owned and look like test domains. I emailed the admins of those domains and am marking this don't merge for a week to let people object.

##### FEATURE FLAG
"REACH: Enable experimental location many-to-many mappings"

### CODE REVIEW

**Review by mkangia — APPROVED** [review]

File: corehq/apps/locations/migrations/0018_remove_related_locations.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.27 on 2020-02-12 22:56
+from __future__ import unicode_literals
```
Comment (esoergel) [inline corehq/apps/locations/migrations/0018_remove_related_locations.py]: nit: outdated `__future__` import

**Review by esoergel — APPROVED** [review]
Very nice.  If ICDS doesn't need it, then I do think it can be removed.

**Comment by orangejenny** [comment]
Closing because the REACH demo app still needs to be maintained.

**Comment by orangejenny** [comment]
Reopening and rebased on master. I'm fairly certain with CAS over, the REACH demo is no longer relevant, but @Rohit25negi @snopoke if you know someone on the old project team team I could/should follow up with, I'd be happy to do so.

**Comment by mkangia** [comment]
@orangejenny 
https://github.com/dimagi/commcare-hq/pull/28896

File: corehq/apps/locations/migrations/0018_remove_related_locations.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        ),
+        migrations.AlterModelManagers(
+            name='sqllocation',
+            managers=[
+            ],
+        ),
+        migrations.DeleteModel(
+            name='LocationRelation',
+        ),
+    ]
```
Comment (snopoke) [inline corehq/apps/locations/migrations/0018_remove_related_locations.py:35]: what does the sql output for this look like?

**Review by snopoke — COMMENTED** [review]
@orangejenny I don't know. @Rohit25negi any thoughts on this?

File: corehq/apps/locations/migrations/0018_remove_related_locations.py (hunk 2 above)
Comment (orangejenny) [reply corehq/apps/locations/migrations/0018_remove_related_locations.py:35]:
```
BEGIN;
--
-- Alter unique_together for locationrelation (0 constraint(s))
--
ALTER TABLE "locations_locationrelation" DROP CONSTRAINT "locations_locationrelati_location_a_id_location_b_e6e38d65_uniq";
--
-- Remove field location_a from locationrelation
--
SET CONSTRAINTS "locations_locationre_location_a_id_3a57268d_fk_locations" IMMEDIATE; ALTER TABLE "locations_locationrelation" DROP CONSTRAINT "locations_locationre_location_a_id_3a57268d_fk_locations";
ALTER TABLE "locations_locationrelation" DROP COLUMN "location_a_id" CASCADE;
--
-- Remove field location_b from locationrelation
--
SET CONSTRAINTS "locations_locationre_location_b_id_d92e664d_fk_locations" IMMEDIATE; ALTER TABLE "locations_locationrelation" DROP CONSTRAINT "locations_locationre_location_b_id_d92e664d_fk_locations";
ALTER TABLE "locations_locationrelation" DROP COLUMN "location_b_id" CASCADE;
--
-- Change managers on sqllocation
--
--
-- Delete model LocationRelation
--
DROP TABLE "locations_locationrelation" CASCADE;
COMMIT;
```

Comment (orangejenny) [reply corehq/apps/locations/migrations/0018_remove_related_locations.py]: Realized there are conflicting migrations, added 59c68cd4b0c to delete the old one and re-run `makemigrations`. Now it's just `DROP TABLE "locations_locationrelation" CASCADE;`

**Comment by orangejenny** [comment]
@mkangia Cool, so this PR will break the ability to restore REACH by reverting just your PR, but at some point that would likely be insufficient anyway, and it should still be possible to check out the commit before your PR. I'm going to merge this in next week.

**Pull request merged**